load-tests/ftp_pubsub/deployment/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
+  - ftp_pubsub.yaml
+  - ingress.yaml
```

Comment:
we should add the deployment scripts of the ftp server here

Reply:
Added the script names.